scopes/ui-foundation/tree/folder-tree-node/folder-tree-node.tsx

```diff
@@ -22,42 +30,20 @@ export function FolderTreeNode({ node, depth }: FolderTreeNodeProps) {
   const [open, setOpen] = useState(node.payload?.open ?? true);
   useEffect(() => {
     // allow node model to override open state
-    node?.payload?.open !== undefined && setOpen(node?.payload?.open)
-  }, [node?.payload?.open])
-  const displayName = node.id.replace(/\/$/, '').split('/').pop();
+    node?.payload?.open !== undefined && setOpen(node?.payload?.open);
+  }, [node?.payload?.open]);
 
-  const CustomIcon = getCustomIcon(node.payload?.icon)
-
-  return (
-    <div>
-      {node.id && (
-        <div className={classNames(indentClass, styles.folder)} onClick={() => setOpen(!open)}>
-          <div className={styles.left}>
-            <Icon className={classNames(styles.icon, !open && styles.collapsed)} of="fat-arrow-down" />
-            {CustomIcon}
-            <span className={styles.name}>{displayName}</span>
-          </div>
-        </div>
-      )}
-      <AnimateHeight height={open ? 'auto' : 0}>
-        <div style={indentStyle(depth + 1)} className={classNames(styles.childrenTree)}>
-          {node.children && <TreeLayer childNodes={node.children} depth={depth + 1} />}
-        </div>
-      </AnimateHeight>
+  const displayName = node.id.replace(/\/$/, '').split('/').pop();
+  const CustomIcon = getCustomIcon(node.payload?.icon);
+  const Title = node.id && (
+    <div className={classNames(indentClass, styles.folder)} onClick={() => setOpen(!open)}>
+      <div className={styles.left}>
+        <Icon className={classNames(styles.icon, !open && styles.collapsed)} of="fat-arrow-down" />
```

Comment:
consider extracting this to a component under design, like `teambit.desgin/ui/surfaces/menu/toggle-bullet`